app/lib/services/voip/MediaSessionInstance.ts

```
@@ -28,6 +28,7 @@ import type { IDDPMessage } from '../../../definitions/IDDPMessage';
 import type { ISubscription, TSubscriptionModel } from '../../../definitions';
 import { getDMSubscriptionByUsername } from '../../database/services/Subscription';
 import { getUidDirectMessage } from '../../methods/helpers/helpers';
+import { Platform, PermissionsAndroid } from 'react-native';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)